### ISSUE TITLE
Running develop branch on scalingo?

### ISSUE THREAD

**Issue by yoonack** [AHAAAAAAA/PokemonGo-Map#2933]
Is it possible to run develop branch on scalingo? Is it integrated in develop branch?
## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used:
* Environment name and version (e.g. Python 2.7):
* Operating System and version (desktop or mobile):

**Comment by 3nvy** [comment]
I would like to know if its possible too, I even tryed to clone the entire project to another repository, change default branch to development and changes repository into scalingo.json file, but still main repo still used :S

**Comment by gopokenc** [comment]
Same issue here - I tried modifying the URL of the repository branch in the deploy button but the master still deploys.

**Comment by johnsudaar** [comment]
Yup you can run develop branch on scalingo. Even if the one click is configured for the master branch (which is working because of the scalingo <-> heroku compatibility).

If you want to run the develop branch just add #develop to the url:

https://my.scalingo.com/deploy?source=https://github.com/AHAAAAAAA/PokemonGo-Map#develop

**Comment by yoonack** [comment]
i tried to run develop branch on scalingo, however I got the error of Grunt build. Is there a way not to run Grunt build for develop branch or is there a way to run grunt on cloud?

**Comment by yoonack** [comment]
Thanks for the help. I tried adding #develop for my own forked repository, however, the page shown on scalingo is still using the one from base repository. I also modified the multipack support as you do, the grunt build error still came out, looks like it's still not using my own repository even after I change the source link

**Comment by johnsudaar** [comment]
Yup, scalingo is using your scalingo.json file to resolve the repository url. And since your scalingo.json is still pointing to this repository your changes will not be seen.

If you want to use the oneclick deploy on your app, you'll need to modify the repository field or your scalingo.json

**Comment by yoonack** [comment]
Thanks for the help. I used your build on scalingo. However, when I deployed it, it stuck at "Deployment is starting...", it just won't proceed. Is there something wrong with the configuration?
